pytest.ini

```diff
@@ -4,6 +4,8 @@
 markers =
     integration: Mark a test as an integration test.  These generally take > 30 seconds to complete.
     smoke: Mark a test as an smoke test. These often exercise the full stack
+    non_default_llm_provider: tests that exercise llm providers other than our default Open AI
```

Comment:
This is important to know about. Say you want to add a new provider, you'll need to add that here, and in your marks within the actual test(s).